### PR TITLE
K@x4l

### PR DESCRIPTION
**Pull request by k0x4l**
hey

### CODE REVIEW

File: src/test/java/day01/test.java [hunk 1]
```diff
@@ -3,6 +3,10 @@
 
 public class test {
     public static void main(String[] args) {
+
```
Comment (silverhillus) [inline src/test/java/day01/test.java:6]: Please review main method

Comment (k0x4l) [reply src/test/java/day01/test.java:6]: I can execute main method

Comment (AhmetSY) [reply src/test/java/day01/test.java:6]: are you sure?

**Comment by AhmetSY** [comment]
hurry up!

**Review by silverhillus — APPROVED** [review]

**Comment by AhmetSY** [comment]
well done!

**Pull request merged**